openedx_events/data.py

```diff
@@ -56,7 +72,8 @@ def __attrs_post_init__(self):
             ),
         )
         object.__setattr__(self, "sourcehost", socket.gethostname())
-        object.__setattr__(self, "time", datetime.utcnow())
         object.__setattr__(
             self, "sourcelib", tuple(map(int, openedx_events.__version__.split(".")))
         )
+        if getattr(self, "time", None) is None:
```

Comment:
Can we use the same default_if_none converter used in minorversion?

Reply:
We cannot. I added a comment to explain.

Reply:
We can, and you helped me learn how.